packages/js/src/ui/components/Header/StatusOptions.tsx

```diff
@@ -16,33 +15,24 @@ export const StatusOptions = (props: { setFeedOptions: (options: FetchFeedArgs)
     <>
       <StatusItem
         label={DropdownStatus.UnreadRead}
-        /**
-         * TODO: Implement setFeedOptions and isSelected after Filter is implemented
-         */
         onClick={() => {
-          props.setFeedOptions({ status: NotificationStatus.UNREAD });
+          props.setFeedOptions({ archived: false });
         }}
         isSelected={true}
         icon={Inbox}
       />
       <StatusItem
         label={DropdownStatus.Unread}
         onClick={() => {
-          /**
-           * TODO: Implement setFeedOptions after Filter is implemented
-           */
-          props.setFeedOptions({ status: NotificationStatus.UNSEEN });
+          props.setFeedOptions({ read: false, archived: false });
```

Comment:
✏ suggestion: when `read: false` then `archived: false` is always true, so we can remove that `archived: false` part